spec/test-outputs/www-integration.out.vcl

```diff
@@ -137,26 +133,33 @@ if (req.http.Cookie ~ "cookies_policy") {
         set req.http.GOVUK-ABTest-Example = "B";
       } else if (req.http.Cookie ~ "ABTest-Example") {
         # Set the value of the header to whatever decision was previously made
-        set req.http.GOVUK-ABTest-Example = req.http.Cookie:ABTest-Example;
-      } else {
-        declare local var.denominator_Example INTEGER;
-        declare local var.denominator_Example_A INTEGER;
-        declare local var.nominator_Example_A INTEGER;
-        set var.nominator_Example_A = std.atoi(table.lookup(example_percentages, "A"));
-        set var.denominator_Example += var.nominator_Example_A;
-        declare local var.denominator_Example_B INTEGER;
-        declare local var.nominator_Example_B INTEGER;
-        set var.nominator_Example_B = std.atoi(table.lookup(example_percentages, "B"));
-        set var.denominator_Example += var.nominator_Example_B;
-        set var.denominator_Example_A = var.denominator_Example;
-        if (randombool(var.nominator_Example_A, var.denominator_Example_A)) {
+        # and is still a known variant
+        if (req.http.Cookie ~ "ABTest-Example=A") {
           set req.http.GOVUK-ABTest-Example = "A";
-        } else {
+        }
+        if (req.http.Cookie ~ "ABTest-Example=B") {
           set req.http.GOVUK-ABTest-Example = "B";
         }
+      } else {
+        # declare local var.denominator_Example INTEGER;
+        # declare local var.denominator_Example_A INTEGER;
+        # declare local var.nominator_Example_A INTEGER;
+        # set var.nominator_Example_A = std.atoi(table.lookup(example_percentages, "A"));
+        # set var.denominator_Example += var.nominator_Example_A;
+        # declare local var.denominator_Example_B INTEGER;
+        # declare local var.nominator_Example_B INTEGER;
+        # set var.nominator_Example_B = std.atoi(table.lookup(example_percentages, "B"));
+        # set var.denominator_Example += var.nominator_Example_B;
+        # set var.denominator_Example_A = var.denominator_Example;
+        # if (randombool(var.nominator_Example_A, var.denominator_Example_A)) {
+        #   set req.http.GOVUK-ABTest-Example = "A";
+        # } else {
+        #   set req.http.GOVUK-ABTest-Example = "B";
+        # }
```

Comment:
I've commented this part out because it relies heavily on Fastly syntax (`declare local var`, `table.lookup`, `randombool`) which we'd need to replace but wasn't necessary in this spike.